gui/wxpython/photo2image/ip2i_manager.py

```diff
@@ -163,7 +164,7 @@ def __init__(
             if p.returncode == 0:
                 print("returncode = ", str(p.returncode))
                 self.Map.region = self.Map.GetRegion()
-        except:
+        except CalledModuleError:
```

Comment:
This is not going to throw CalledModuleError, I am not sure though which error this should catch, so use Exception.